includes/Admin/Dashboard.php

```diff
@@ -423,6 +423,7 @@ public function get_dashboard_settings(): array {
 				'stories'        => trailingslashit( $this->story_post_type->get_rest_url() ),
 				'media'          => '/web-stories/v1/media/',
 				'currentUser'    => '/web-stories/v1/users/me/',
+				'fonts'          => '/web-stories/v1/fonts/',
```

Comment:
We should do this 

https://github.com/google/web-stories-wp/blob/3824d2cee3e231ae0795d3b08f6eb15417f49399/includes/Admin/Editor.php#L404